pyproject.toml

```diff
@@ -35,7 +35,7 @@ include = ["*"]
 
 [tool.black]
 line-length = 120
-target-version = ['py38', 'py39', 'py310', 'py311', 'py312']
+target-version = ['py38', 'py39', 'py310', 'py311']
```

Comment:
How comes you are removing this?

Reply:
I get this weird error when it's included:
```bash
Error: Invalid value for '-t' / '--target-version': 'py312' is not one of 'py33', 'py34', 'py35', 'py36', 'py37', 'py38', 'py39', 'py310', 'py311'.
```

Reply:
Interesting.
Maybe old version of black?

Reply:
Nope, latest pypi release 24.8.0 and github dev version have the same problem...

Reply:
OK, let's remove this:
https://github.com/catalystneuro/cookiecutter-my-lab-to-nwb-template/pull/28